demo/site/src/app/block-preview/page/page.tsx

```diff
@@ -3,17 +3,17 @@ import { BlockPreviewProvider, IFrameBridgeProvider, useBlockPreviewFetch, useIF
 import { PageContentBlockData } from "@src/blocks.generated";
 import { PageContentBlock } from "@src/blocks/PageContentBlock";
 import { recursivelyLoadBlockData } from "@src/recursivelyLoadBlockData";
-import { graphQLApiUrl } from "@src/util/graphQLClient";
 import { useEffect, useState } from "react";
 
 const PreviewPage = () => {
     const iFrameBridge = useIFrameBridge();
 
-    const { fetch, graphQLFetch } = useBlockPreviewFetch(graphQLApiUrl);
+    const { fetch, graphQLFetch } = useBlockPreviewFetch();
```

Comment:
I just tested this: This is a breaking change. If I provide an argument, TS complains:

<img width="885" alt="image" src="https://github.com/user-attachments/assets/52685aa6-cb47-4315-a592-d8ae66f809db">

Reply:
"Zwei Fliegen mit einer Klappe": https://github.com/vivid-planet/comet/pull/2826